integrations/kube-agent-updater/go.mod

```diff
@@ -10,11 +10,11 @@ require (
 	github.com/jonboulle/clockwork v0.4.0
 	github.com/opencontainers/go-digest v1.0.0
 	github.com/sigstore/cosign/v2 v2.0.1-0.20230228132138-2f1ec646de33
-	github.com/sigstore/sigstore v1.6.3
+	github.com/sigstore/sigstore v1.6.4
```

Comment:
kube-agent-updater bumps cause a bunch of problems, we should likely drop them for now and let @hugoShaka do the bump.